cmd/earth/main.go

```diff
@@ -1450,6 +1462,42 @@ func (app *earthApp) newBuildkitdClient(ctx context.Context, opts ...client.Clie
 	return bkClient, nil
 }
 
+func (app *earthApp) collectAnalytics(exitCode int, realtime time.Duration) {
+	if app.cfg.Global.DisableAnalytics {
+		return
+	}
+	installID, ci := os.LookupEnv("CI")
+	if !ci {
+		var err error
+		installID, err = getInstallID()
+		if err != nil {
+			installID = "unknown"
+		}
+	}
+	segmentClient := analytics.New("RtwJaMBswcW3CNMZ7Ops79dV6lEZqsXf")
+	segmentClient.Enqueue(analytics.Track{
+		Event:  "exit",
```

Comment:
Reading a little more on the amplitude docs, I think it's better to use the command name here, as the event (possibly with `"earth "` as a prefix), because this is the actual action that the user performed.

Reply:
sounds like a good change, I was hoping we could configure it somewhere in the pipeline but perhaps not.